core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -1307,6 +1307,7 @@ angular.module('oppia').directive('conversationSkin', [
           };
 
           ctrl.$onInit = function() {
+            questionPlayerConfig = $scope.getQuestionPlayerConfig();
```

Comment:
Currently `questionPlayerConfig` is being initialized in the constructor.
Which makes it hard to test the parity checks.
So, I've moved this logic to $onInit.